protobuf/src/stream.rs

```diff
@@ -31,6 +31,9 @@ use buf_read_iter::BufReadIter;
 // `CodedOutputStream` wraps `BufWriter`, it often skips double buffering.
 const OUTPUT_STREAM_BUFFER_SIZE: usize = 8 * 1024;
 
+// Default recursion level limit.
+const DEFAULT_RECURSION_LIMIT: isize = 100;
```

Comment:
100 is just copied from [C++ implementation](https://github.com/google/protobuf/blob/master/src/google/protobuf/io/coded_stream.cc#L82). If you think it's too small, please let me know.

Reply:
Thanks for the explanation. Could you please paste the explanation into the source comments?